src/components/PaginatedTable/Customize/CellView.js

```
@@ -0,0 +1,287 @@
+          {caseId}
+        </Typography>
+      </Link>
+      {
+        ((!unifiedView) && hasMultiStudyParticipants(otherCases))
+        && (
+          <>
+            <MultiStudyTooltip
+              tableMeta={otherCases}
+              value={caseId}
+            />
+          </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)